tests/e2e/specs/Modeler.spec.js

```diff
@@ -0,0 +1,216 @@
+        { x: coordinateGenerator(), y: coordinateGenerator()},
+      );
+    });
+    cy.get('.modeler').children().should('have.length', 8);
+  });
+
+  it('Creates top and bottom lane', () => {
+    const poolSelector = '#v-24';
+    const topLaneSeletor = '#v-26';
+    const bottomLaneSeletor = '#v-29';
+
+    dragFromSourceToDest(
+      '.processmaker-modeler-pool',
+      '.paper-container',
+      { x: 100, y: 100},
+    );
+    cy.get(poolSelector).click();
+    cy.get(topLaneSeletor).click({ force: true });
+    cy.get(bottomLaneSeletor).click({ force: true });
+
+  });
+
+  it('Update task name', ()=> {
+    dragFromSourceToDest(
+      '.processmaker-modeler-task',
+      '.paper-container',
+      { x: coordinateGenerator(), y: coordinateGenerator()},
+    );
+
+    cy.get('.joint-viewport').find('.joint-type-processmaker-components-nodes-task').click({force: true});
+    typeIntoTextInput('[name=\'name\']', 'testing');
+    cy.get('[name=\'name\']').should('have.value', 'testing');
+  });
+
+  it('Update exclusive gateway name', ()=> {
+    dragFromSourceToDest(
+      '.processmaker-modeler-exclusive-gateway',
+      '.paper-container',
+      { x: coordinateGenerator(), y: coordinateGenerator()},
+    );
+
+    cy.get('.joint-viewport').find('.joint-type-processmaker-modeler-bpmn-exclusivegateway').click({force: true});
+    typeIntoTextInput('[name=\'name\']', 'testing');
+    cy.get('[name=\'name\']').should('have.value', 'testing');
+  });
+
+  it('Update text annotation name', ()=> {
+    dragFromSourceToDest(
+      '.processmaker-modeler-text-annotation',
+      '.paper-container',
+      { x: coordinateGenerator(), y: coordinateGenerator()},
+    );
+
+    cy.get('.joint-viewport').find('.joint-type-standard-polyline').click({force: true});
+    typeIntoTextInput('[name=\'text\']', 'testing');
+    cy.get('[name=\'text\']').should('have.value', 'testing');
+  });
+
+  it('Update pool name', ()=> {
+    dragFromSourceToDest(
+      '.processmaker-modeler-pool',
+      '.paper-container',
+      { x: coordinateGenerator(), y: coordinateGenerator()},
+    );
+
+    cy.get('.joint-viewport').find('.joint-type-processmaker-modeler-bpmn-pool').click({force: true});
+    typeIntoTextInput('[name=\'name\']', 'testing');
+    cy.get('[name=\'name\']').should('have.value', 'testing');
+  });
+
+  it('Create a simple process', () => {
+    const startEventSelector = '#v-11';
+    const startEventConnectorSelector = '#v-14';
+    const taskSelector = '#v-23';
+    const taskConnectorSelector = '#v-26';
+    const taskSelectorTwo = '#v-47';
+    const taskConnectorSelectorTwo = '#v-50';
+    const taskSelectorThree = '#v-68';
+    const taskConnectorSelectorThree = '#v-71';
+    const endEventSelector = '#v-85';
+    const poolSelector = '#v-101';
+
+    cy.get('.modeler').children().should('have.length', 2);
+
+    dragFromSourceToDest(
+      '.processmaker-modeler-task',
+      '.paper-container',
+      { x: 300, y: 200},
```

Comment:
Not using `coordinateGenerator` for these? :p

Reply:
I use the `x, y` of the shape to connect other shapes. If we used to `generateRandomPoint()` here we would not know what the `x, y` would be for the connection.